web-ng/src/app/app.component.ts

```diff
@@ -29,8 +29,11 @@ import { environment } from '../environments/environment';
 })
 export class AppComponent {
   public constructor(@Inject(DOCUMENT) private doc: Document) {
-    this.initGoogleMap();
+    if (environment.googleMapsApiKey !== '') {
```

Comment:
Please add a comment indicating this is for testing

Reply:
Added an `env` entry in environment variables, and used that.